src/behaviours/material/index.js

```diff
@@ -33,6 +33,12 @@ const Material = (ref, jsClass, watchedProp) => Component => class MaterialCompo
             Component.prototype.componentWillReceiveProps.call(this, nextProps);
         }
     }
+
+    componentDidUpdate() {
+        if(jsClass === 'MaterialTextfield') {
+            this.refs.inputText.MaterialTextfield.change(this.props.value);
```

Comment:
@GuenoleK It seems strange that there is only one class which triggers something.